<launch>
    <param name = "x_left" type = "double" value = "0.05" />
    <param name = "x_right" type = "double" value = "15" />
    <param name = "y_left" type = "double" value = "-3" />
    <param name = "y_right" type = "double" value = "3" />
    <param name = "point_min" type = "double" value = "5" />
    <param name = "point_max" type = "double" value = "1000" />

    <node name="cluster_disorder" pkg = "rfans_driver" type = "cluster_disorder"  />
    <node name="guo3" pkg = "rfans_driver" type = "guo3" output = "screen" />
    <node name="rviz" pkg = "rviz" type = "rviz" output = "screen" />
</launch>
